src/reference/asciidoc/kafka.adoc

```diff
@@ -2,6 +2,38 @@
 [[kafka]]
 === Using Spring for Apache Kafka
 
+==== Configuring Topics
+
+If you define a `KafkaAdmin` bean in your application context, it can automatically add topics to the broker.
+Simply add a `NewTopic` `@Bean` for each topic to the application context.
+
+[source, java]
+----
+@Bean
+public KafkaAdmin admin() {
+    Map<String, Object> configs = new HashMap<>();
+    configs.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG,
+            StringUtils.arrayToCommaDelimitedString(kafkaEmbedded().getBrokerAddresses()));
+    return new KafkaAdmin(configs);
+}
+
+@Bean
+public NewTopic topic1() {
+    return new NewTopic("foo", 10, (short) 2);
```

Comment:
This is silly to have such a high-level API requiring the cast to `short`, but I think we don't have choice and will live with this like with all the inconvenience in Kafka 😄